commit 88d602f4a45ffcaf8fc7b0cda132ceaaed264556
Author: Hugo Osvaldo Barrera <hugo@whynothugo.nl>
Date:   Tue Apr 15 16:49:32 2025 +0200

    Add support for Linux on aarch64 and armv7
    
    Fixes: https://github.com/jacktrip/jacktrip/issues/1404
    Patch-Source: https://github.com/jacktrip/jacktrip/pull/1405

diff --git a/externals/oscpp/include/oscpp/detail/endian.hpp b/externals/oscpp/include/oscpp/detail/endian.hpp
index f9a0c2e0..bdb061c1 100644
--- a/externals/oscpp/include/oscpp/detail/endian.hpp
+++ b/externals/oscpp/include/oscpp/detail/endian.hpp
@@ -71,7 +71,8 @@
     defined(__ia64__) || defined(_M_IX86) || defined(_M_IA64) ||     \
     defined(_M_ALPHA) || defined(__amd64) || defined(__amd64__) ||   \
     defined(_M_AMD64) || defined(__x86_64) || defined(__x86_64__) || \
-    defined(_M_X64) || defined(__bfin__)
+    defined(_M_X64) || defined(__bfin__) || defined(__aarch64__) || \
+    defined(__ARM_EABI__)
 
 #    define OSCPP_LITTLE_ENDIAN
 #    define OSCPP_BYTE_ORDER OSCPP_BYTE_ORDER_LITTLE_ENDIAN
